docs/index.rst

```diff
@@ -94,6 +94,14 @@ install <https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs>
 
     resource/**
 
+.. toctree::
```

Comment:
I couldn't get sphinx to run without warning without adding this, as well as http.rst. asgi accepts a param of type ExcludedPaths which comes from utils, and sphinx would not compile docs for that field without docs for utils.http existing as well.